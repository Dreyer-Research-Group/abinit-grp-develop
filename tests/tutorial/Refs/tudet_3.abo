
.Version 9.3.1 of UJDET 
.(MPI version, prepared for a x86_64_linux_gnu9.3 computer) 

.Copyright (C) 1998-2021 ABINIT group . 
 UJDET comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read https://docs.abinit.org/theory/acknowledgments for suggested
 acknowledgments of the ABINIT effort.
 For more information, see https://www.abinit.org .

.Starting date : Tue 10 Nov 2020.
- ( at 22h24 )
  

 # input for ujdet, cut it using 'sed -n "/MARK/,/MARK/p" abi.out > ujdet.in '------- 

           ndtset        4
              nat        2
           nspden        2
         macro_uj        1
           pawrad      2.11531574E+00
        ph0phiint      8.68772655E-01
        pawprtvol        0
              vsh1     3.6749325398E-03  3.6749325398E-03  0.0000000000E+00
                       0.0000000000E+00
              vsh2     3.6749325398E-03  3.6749325398E-03  0.0000000000E+00
                       0.0000000000E+00
              vsh3    -3.6749325398E-03 -3.6749325398E-03 -0.0000000000E+00
                      -0.0000000000E+00
              vsh4    -3.6749325398E-03 -3.6749325398E-03 -0.0000000000E+00
                      -0.0000000000E+00
              occ1     4.2987910665E+00  1.7583919826E+00  4.3105733757E+00
                       1.8111689962E+00
              occ2     4.3071021199E+00  1.7756215188E+00  4.3022496797E+00
                       1.7935831525E+00
              occ3     4.3234112772E+00  1.8333344036E+00  4.2957721122E+00
                       1.7449851039E+00
              occ4     4.3074343044E+00  1.7982435946E+00  4.3122107740E+00
                       1.7803773646E+00
             xred      0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
                       5.0000000000E-01  5.0000000000E-01  5.0000000000E-01
           rprimd      5.4200000000E+00  0.0000000000E+00  0.0000000000E+00
                       0.0000000000E+00  5.4200000000E+00  0.0000000000E+00
                       0.0000000000E+00  0.0000000000E+00  5.4200000000E+00
            scdim           6       6       6
            wfchr      4.3718854923E+00  3.0000000000E+00  2.0000000000E+00
                       1.4456967778E+00 -7.3619476684E-01  1.1428505601E-01

 # further possible options: 
 #    scdim    2 2 2 
 #    mdist    10.0  
 #  pawujga    2 
 # pawujopt    2 
 # pawujrad    3.0
 # ------- end input for ujdet: end-MARK  -------- 


 ---------- calculate U, (J) start ---------- 

 All pawujat  ok and equal to   1
 All macro_uj ok and equal to   1
 URES      ii    nat       r_max    U(J)[eV]   U_ASA[eV]   U_inf[eV]
 URES       1      2     4.69390     3.86230     3.10778     2.71714
 URES       2     16     9.38770     7.28001     5.85781     5.12150
 URES       3     54    14.08160     7.60740     6.12125     5.35182
 URES       4    128    18.77540     7.67626     6.17665     5.40026
 URES       5    250    23.46930     7.69849     6.19454     5.41590
 URES       6    432    28.16310     7.70781     6.20204     5.42246

 ---------- calculate U, (J) end -------------- 

