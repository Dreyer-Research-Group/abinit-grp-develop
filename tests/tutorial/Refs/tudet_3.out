
.Version 8.0.3 of UJDET 
.(MPI version, prepared for a x86_64_linux_gnu5.3 computer) 

.Copyright (C) 1998-2018 ABINIT group . 
 UJDET comes with ABSOLUTELY NO WARRANTY.
 It is free software, and you are welcome to redistribute it
 under certain conditions (GNU General Public License,
 see ~abinit/COPYING or http://www.gnu.org/copyleft/gpl.txt).

 ABINIT is a project of the Universite Catholique de Louvain,
 Corning Inc. and other collaborators, see ~abinit/doc/developers/contributors.txt .
 Please read ~abinit/doc/users/acknowledgments.html for suggested
 acknowledgments of the ABINIT effort.
 For more information, see https://www.abinit.org .

.Starting date : Mon  4 Apr 2016.
- ( at 09h45 )
  

 # input for ujdet, cut it using 'sed -n "/MARK/,/MARK/p" abi.out > ujdet.in '------- 

           ndtset        4
              nat        2
           nspden        2
         macro_uj        1
           pawrad      2.02984247E+00
        ph0phiint      8.54332879E-01
        pawprtvol        0
              vsh1     3.6749325398E-03  3.6749325398E-03  0.0000000000E+00
                       0.0000000000E+00
              vsh2     3.6749325398E-03  3.6749325398E-03  0.0000000000E+00
                       0.0000000000E+00
              vsh3    -3.6749325398E-03 -3.6749325398E-03 -0.0000000000E+00
                      -0.0000000000E+00
              vsh4    -3.6749325398E-03 -3.6749325398E-03 -0.0000000000E+00
                      -0.0000000000E+00
              occ1     4.1143258210E+00  1.6703524785E+00  4.1251256651E+00
                       1.7123048039E+00
              occ2     4.1220490763E+00  1.6839286426E+00  4.1179968737E+00
                       1.6987556741E+00
              occ3     4.1372361585E+00  1.7304565859E+00  4.1122386749E+00
                       1.6606834131E+00
              occ4     4.1224155422E+00  1.7026241698E+00  4.1264239301E+00
                       1.6878975678E+00
             xred      0.0000000000E+00  0.0000000000E+00  0.0000000000E+00
                       5.0000000000E-01  5.0000000000E-01  5.0000000000E-01
           rprimd      5.4200000000E+00  0.0000000000E+00  0.0000000000E+00
                       0.0000000000E+00  5.4200000000E+00  0.0000000000E+00
                       0.0000000000E+00  0.0000000000E+00  5.4200000000E+00
            scdim           6       6       6
            wfchr      4.3772661697E+00  3.0000000000E+00  2.0000000000E+00
                       1.4530610236E+00 -7.4483924347E-01  1.1680994649E-01

 # further possible options: 
 #    scdim    2 2 2 
 #    mdist    10.0  
 #  pawujga    2 
 # pawujopt    2 
 # pawujrad    3.0
 # ------- end input for ujdet: end-MARK  -------- 


 ---------- calculate U, (J) start ---------- 

 All pawujat  ok and equal to   1
 All macro_uj ok and equal to   1
 URES      ii    nat       r_max    U(J)[eV]   U_ASA[eV]   U_inf[eV]
 URES       1      2     4.69390     4.74555     3.67983     3.20150
 URES       2     16     9.38770     8.77694     6.80588     5.92122
 URES       3     54    14.08160     9.17082     7.11130     6.18694
 URES       4    128    18.77540     9.25647     7.17772     6.24472
 URES       5    250    23.46930     9.28509     7.19991     6.26403
 URES       6    432    28.16310     9.29738     7.20944     6.27232

 ---------- calculate U, (J) end -------------- 

